ga4gh/_protocol_definitions.py

```diff
@@ -3071,25 +3070,25 @@ def __init__(self, **kwargs):
 postMethods = \
     [('/callsets/search',
       SearchCallSetsRequest,
-      SearchCallSetsResponse),
+      SearchReadsResponse),
```

Comment:
Hm, couldn't reproduce this.  I'll create a PR with your changes minus the erroneous diff in postMethods.